packages/ui-app/src/@types/file-saver/types.d.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+// https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html
+declare module 'file-saver';
```

Comment:
If these definitions are not available on definately-typed, they need to be added to the polkadot-js/ts repo.